web/fotorama/fotorama.js

```diff
@@ -3218,12 +3218,18 @@ fotoramaVersion = '4.6.4';
         }
 
         that.cancelFullScreen = function () {
+            const fotoramaFramesCollection = document.getElementsByClassName('fotorama__stage__frame')
+
             if (o_nativeFullScreen && fullScreenApi.is()) {
                 fullScreenApi.cancel(document);
             } else {
                 cancelFullScreen();
             }
 
+            for (let element of fotoramaFramesCollection) {
+                element.style.opacity = '0'
+            }
+
```

Comment:
Oh I didn't know that fotorama.js is overwritten in alpaca theme :open_mouth: 

Maybe it would be better to use for..in loop for better browser compability
Please also check if `fotoramaFramesCollection` can be undefined or empty and add proper conditions if so

Reply:
Please also check if fotoramaFramesCollection can be undefined or empty and add proper conditions if so -> 👍 
There is no babel, I think we should use jus `for` loop here.  Please check IE